config/co2eq_parameters.js

```diff
@@ -60,9 +60,10 @@ exports.renewableAccessor = (zoneKey, k, v) => {
 
 exports.defaultPowerOriginRatio = (zoneKey, fuelKey) => {
   //if no ratios found for that zoneKey, use defaults
-  const key = (co2eqParameters.fallbackZoneMixes[zoneKey] || {}).powerOriginRatios ? zoneKey : 'defaults' ;
-  
-  return co2eqParameters.fallbackZoneMixes[key].powerOriginRatios[fuelKey]
+  const ratio = (co2eqParameters.fallbackZoneMixes.zoneOverrides[zoneKey] || 
```

Comment:
you could return directly at this line instead of declaring a variable and returning it after